### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by dolciss** [mozzius/graysky#115] [Feature requests]
Are there any plans for translations into other languages?
There are several issues and pull requests on the official Bluesky app, but it doesn't look like it will be supported yet.
cf. social-app#782, 1720, 1740

I would change it if I could, but I am new to react native and am having difficulty.
If you have a list of words to translate, I would like to translate them into Japanese (I am Japanese).

**Comment by mozzius** [comment]
I would love to add this!

**Comment by dolciss** [comment]
The #412 by reindex-ot triggered the implementation in #416.
So this discussion is closed, thank you!
